packages/react-components/src/components/utils/GridLayoutUtils.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BlockProps } from '../GridLayout';
+
+const TARGET_RATIO = 16 / 9;
+const TOLERANCE_RATIO = 8 / 9;
+
+const isRatioBetterThan = (A: number, B: number, targetRatio?: number): boolean => {

```

Comment:
you can pass TARGET_RATIO as a default function parameter
`const isRatioBetterThan = (A: number, B: number, targetRatio = TARGET_RATIO): boolean => {`
you can get rid of line 10 then

Reply:
What is the deciding factor for which one is better?
From what I can understand, its the one that is closest to the targetRatio i.e., leaves the least amount of whitespace.
Can we document this?